docs-source/spring/content/development/project.md

```diff
@@ -0,0 +1,205 @@
+* Java 17 or 21
+
+### Dependencies
+
+Oracle recommends you add the following dependencies to your application so that it
+can take advantage of the built-in platform services:
+
+```xml
+<properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <java.version>17</java.version>
+    <spring.boot.dependencies.version>3.1.5</spring.boot.dependencies.version>
+    <spring-cloud.version>2022.0.4</spring-cloud.version>
+</properties>
+
+<dependencies>
+    <dependency>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-web</artifactId>
+    </dependency>
+    <dependency>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-actuator</artifactId>
+    </dependency>
+    <dependency>
+        <groupId>io.micrometer</groupId>
+        <artifactId>micrometer-registry-prometheus</artifactId>
+    </dependency>
+    <dependency>
+        <groupId>org.springframework.cloud</groupId>
+        <artifactId>spring-cloud-starter-netflix-eureka-client</artifactId>
+    </dependency>
+</dependencies>
+
+<dependencyManagement>
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-dependencies</artifactId>
+            <version>${spring.boot.dependencies.version}</version>
+            <type>pom</type>
+            <scope>import</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.cloud</groupId>
+            <artifactId>spring-cloud-dependencies</artifactId>
+            <version>${spring-cloud.version}</version>
+            <type>pom</type>
+            <scope>import</scope>
+        </dependency>
+    </dependencies>
+</dependencyManagement>    
+```
+
+### Spring Application Configuration
+
+Oracle recommends the following configuration to configure the application access to
+use the built-in services, including the Spring Eureka Service Registry and the
+observability tools.
+
+```yaml
+spring:
+  application:
+    name: account
+  zipkin:
+    base-url: ${zipkin.base-url}
+
+eureka:
+  instance:
+    hostname: ${spring.application.name}
+    preferIpAddress: true
+  client:
+    service-url:
+      defaultZone: ${eureka.service-url}
+    fetch-registry: true
+    register-with-eureka: true
+    enabled: true
+    
+management:
+  endpoint:
+    health:
+      show-details: always
+  endpoints:
+    web:
+      exposure:
+        include: "*"
+  metrics:
+    tags:
+      application: ${spring.application.name}
+```
+
+The variables in this configuration will be automatically injected to your deployment
+and pods when you use the Oracle Backend for Spring Boot and Microservices CLI to deploy your application. 
+
+#### Data Sources
+
+If your application uses a data source, you should add the following configuration.  Note that this
+example shows JPA, if you are using JDBC you should use the appropriate configuration.
```

Comment:
"Note that this example shows the Java Persistence API (JPA)". If you are using JDBC, then use the appropriate configuration:"